app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -927,8 +939,7 @@ private boolean shouldShowComments() {
     //////////////////////////////////////////////////////////////////////////*/
 
     private void openBackgroundPlayer(final boolean append) {
-        AudioStream audioStream = currentInfo.getAudioStreams()
-                .get(ListHelper.getDefaultAudioFormat(activity, currentInfo.getAudioStreams()));
+        AudioStream audioStream = getDefaultAudioStream();
```

Comment:
This and 2 lines below could be `final`

Reply:
I updated it. This line and the next one were not final in the original code though.